liveview-native/liveview-client-swiftui
# Livebook URL Improvements

# **Livebook URL Improvements**
* Fixed the location of the run in livebook urls
* Changed image urls to use IMGUR instead of the documentation_assets project (which can be closed unless it's used in other projects)

**unrelated**
* Removed unused old notebooks

## How to Test
To test this PR generate the documentation locally. You can skip gen docs to save time:
`mix docs --skip-gen-docs=true

Open the `index.html` file in your browser. For me, that's `/Users/brooklin/liveview-client-swiftui/doc/index.html`.

See the generated documentation. 

### **To test the run in Livebook button**
open the "Create a SwiftUI Application" guide under the "Livebooks" section on hex then press the "Run in Livebook Button". This should open the Livebook on the Livebook dev runner site.

### **To test the images**
Open the guide(s) with image changes. Ensure the images are found and match the original image.
